### ISSUE TITLE
Change password is not possible

### ISSUE THREAD

**Issue by anastasiyaig** [status-im/status-desktop#3426]
1. open profile
2. privacy and security
3. change password
4. enter new password and confirm

```
DBG 2021-09-08 15:25:49+03:00 Verifying mailserver connection state...   topics="mailserver model" tid=2213901 file=model.nim:196
ReEncryptKeyStoreDir error: walk error: unable to re-encrypt key file: decryption error: could not decrypt key with given password, path: /Users/anastasiya/data/StatusBeta12/data/keystore/UTC--2021-09-08T10-00-16.479095000Z--e42caf49b44c4706ba4c29a55187af2cb46f6877, name: UTC--2021-09-08T10-00-16.479095000Z--e42caf49b44c4706ba4c29a55187af2cb46f6877
```

<img width="1344" alt="Screenshot 2021-09-08 at 15 26 57" src="https://user-images.githubusercontent.com/82375995/132508995-24ee5c01-9d4c-41fa-8410-f1192476d286.png">

**Comment by pinebit** [comment]
based on the error, the current password seems to be wrong...

**Comment by anastasiyaig** [comment]
how do u mean 

> based on the error, the current password seems to be wrong...

since i am able to login to the app using the password @pinebit ?

**Comment by pinebit** [comment]
> how do u mean
> 
> > based on the error, the current password seems to be wrong...
> 
> since i am able to login to the app using the password @pinebit ?

Just wanted to make sure you enter the current password correct. Otherwise the error looks terrible, maybe password hashing rules has changed, I will investigate this.

**Comment by anastasiyaig** [comment]
feel free to reproduce the issue yourself @pinebit to be 100% sure.

**Comment by pinebit** [comment]
@anastasiyaig I just tested this on the latest master (`8f9c46b`) and it worked fine.

**Comment by anastasiyaig** [comment]
try `Version: 0.3.0-alpha.3` @pinebit .

**Comment by pinebit** [comment]
as per @anastasiyaig 
> change password works fine in the branch with fix of #3412
shall not we close this ticket?
